src/core/Table/Table.tsx

```diff
@@ -274,6 +323,9 @@ export const Table = <
       case TableActions.setFilter:
         onFilterHandler(newState, action, previousState, instance);
         break;
+      case TableActions.toggleRowExpanded:
```

Comment:
Add `toggleAllRowsExpanded` too if someone decides to manually make expand all header column.